test/core/tsi/test_creds/crl_data/current.crl

```diff
@@ -1,12 +1,15 @@
 -----BEGIN X509 CRL-----
-MIIB2TCBwgIBATANBgkqhkiG9w0BAQsFADBWMQswCQYDVQQGEwJBVTETMBEGA1UE
-CAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRk
-MQ8wDQYDVQQDDAZ0ZXN0Y2EXDTIyMDIwNDIyMTI1MFoXDTMyMDIwMjIyMTI1MFow
-JzAlAhRSILpknGPXIWdLvLVIY7DlId/IQRcNMjIwMjA0MjIxMTExWqAPMA0wCwYD
-VR0UBAQCAhABMA0GCSqGSIb3DQEBCwUAA4IBAQAZXNfxSjT/EZDTGV71eE0jKKsg
-Ur8TNkRGypZXbV8cQ+YFlqt2Zp+dsWEP2FLsc048QGVe4sRuJrPOm7eSmvgZUHSX
-l1yI2T6si1wxhX2DKIKDZGWWYx6rOyocL9EjhxZjLSeJ43eLxzD6TnGE29cbDLXv
-bs9slsGyc+UZaD9KY9RpeJjQEV0Yh7+iwIG3PBVKNSFR2R2m8XK+Ioc5Z7zsWwZu
-7gCZ/CW/07/SXfFxM6Q8XTRFStIjqb3cuwivc+ig/X+RbpxDygy0SQA/oJXuABIh
-0y4+YxRUQauODK4vxLPeC1m7tHyh1poVY/jnp3tAn3ax6uFbwDz1wbNtTOeT
+MIICUDCCATgCAQEwDQYJKoZIhvcNAQELBQAwVjELMAkGA1UEBhMCQVUxEzARBgNV
```

Comment:
This file is not exported in BUILD. Is it redundant?

Reply:
Yes - `openssl rehash` creates a symlink to this, and the symlink is exported in BUILD

Reply:
To explain further, this was already in the repo. The generation script copies it around to different sub folders that hold different CRLs, then I run rehash on each of those folders (it's near the end of `intermediate_gen.sh`)

Reply:
Makes sense. Thanks!